GherkinSyncTool/README.md

```diff
@@ -0,0 +1,90 @@
+# GherkinSyncTool
+
+GherkinSyncTool is an open-source console application, that synchronizes tests scenarios in
+a [gherkin syntax](https://cucumber.io/docs/gherkin/) also known as feature files with a tests management system or any
+other destinations. Currently, a TestRail synchronizer is implemented
+
+## Installation
+
+- Install [.NET](https://dotnet.microsoft.com/download)
+- Build solution
+```
+dotnet build
+```
+- Run app
+```
+dotnet .\.\GherkinSyncTool\bin\Debug\net5.0\GherkinSyncTool.dll
+```
+
+## Configuration
+
+GherkinSyncTool could be configured in three ways. The priority corresponds to the list order.
+
+1. appsettings.json
+2. Environment variables
+3. Command-line arguments
+
+### TestRail
+
+For the correct work of the tool, the TestRail template should have custom fields, which are presented in the table below. The template should not contain the required fields.
+
+|System Name|Type|
+|----|----|
+|`preconds`| Text type|
+|`steps_separated`| Step type|
+|`custom_tags`| String type|
+
+### Common configuration
+
+|Parameter|Description|Required|
+|----|----|----|
+|BaseDirectory|Absolute or relevant path to a folder that contains *.feature files|Yes|
+|TagIdPrefix|A tag prefix that will be used for mark test scenarios as synchronized with a test management system|Yes|
+
+### TestRail configuration
+
+|Parameter|Description|Required|
+|----|----|----|
+|TestRailProjectId|Id of a project that will be used for synchronization|Yes|
+|TestRailSuiteId|Id of a suite that will be used as a parent for creating sections|Yes|
+|TestRailTemplateId|Id od a template that will be used for creating or updating test cases|Yes|
```

Comment:
I think it should be "Id oR a template"

Reply:
I suppose of

Reply:
I believe it is spelling error, should be "of" instead of "od"

Reply:
Fixed